File: fixtures/scale/dart/large/lib/ciphers/adler32/adler32_vectors.dart
/// Fixed plaintext corpus the adler32 suite round-trips and hashes.
List<String> adler32Samples() => [
  'HOW VEXINGLY QUICK DAFT ZEBRAS JUMP',
  'BRIGHT VIXENS JUMP DOZY FOWL QUACK',
  'JACKDAWS LOVE MY BIG SPHINX OF QUARTZ',
];

int adler32SampleCount() => adler32Samples().length;

File: fixtures/scale/java/large/src/cryptobench/ciphers/adler32/Adler32Vectors.java
package cryptobench.ciphers.adler32;

import java.util.List;

public class Adler32Vectors {
    public static List<String> samples() {
        return List.of(
            "HOW VEXINGLY QUICK DAFT ZEBRAS JUMP",
            "BRIGHT VIXENS JUMP DOZY FOWL QUACK",
            "JACKDAWS LOVE MY BIG SPHINX OF QUARTZ");
    }
}
